solt/transforms/_transforms.py

```diff
@@ -589,6 +583,31 @@ def _apply_img(self, img: np.ndarray, settings: dict):
     def _apply_mask(self, mask: np.ndarray, settings: dict):
         return self._apply_img_or_mask(mask, settings)
 
+    def _apply_img_or_mask_pt(self, img_mask: torch.Tensor, settings: dict):
```

Comment:
I feel like the implementation of this function is not much different for 2D or ND. Should we call `_apply_img_or_mask_nd` for both opencv and torch backends, a and then via simple `if` detect which backend we should use?